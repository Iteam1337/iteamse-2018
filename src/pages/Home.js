// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/HomePage'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import iteamStylized from './img/iteam_stylized.png'
import Header from '../components/Header/Header'
import Team from '../components/Team/Team'
import GridColumn from '../components/Grid/GridColumn'
import PaddedRow from '../components/Grid/PaddedRow'
import OperationsBanner from '../components/Banners/OperationsBanner'
import H1 from '../components/Typography/H1'
import Paragraph from '../components/Typography/Paragraph'
import Link from '../components/Link/Link'
import styled from 'styled-components'
import MediaQuery from 'react-responsive'
import { media } from '../theme'

type Props = Iteam.ApolloBase<IteamCMS.HomePage>

export const HomePageQuery = gql`
  query HomePage {
    pageStart {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      codeMobileImage
      codeText
      codeTitle
      codeLogo
      codeSlug
      codeLinkText
      cultureMobileImage
      cultureText
      cultureTitle
      cultureLogo
      cultureSlug
      cultureLinkText
      strategyMobileImage
      strategyText
      strategyTitle
      strategyLogo
      strategySlug
      strategyLinkText
    }
  }
`

const HomeGridColumn = GridColumn.extend`
  background-color: ${({ theme }) => theme.colors.concrete};
`

const Content = styled.div`
  display: grid;
  grid-column-gap: 40px;
  grid-template-columns: 1fr;
  padding-bottom: 40px;
  padding-top: 40px;

  ${media.desktop`
    grid-template-columns: 550px 1fr;
    padding-bottom: 100px;
    padding-top: 100px;
  `};
`

const Texts = styled.div`
  align-items: center;
  display: grid;
  grid-row-gap: 40px;
`

const TextBlock = PaddedRow.extend`
  &:not(:last-child) {
    margin-bottom: 40px;
  }

  ${media.desktop`
    &:not(:last-child) {
      margin-bottom: 0;
    }
  `};
`

const StylizedIteam = styled.img`
  display: none;

  ${media.desktop`
    display: block;
  `};
`

const CompanyLogo = styled.img`
  display: block;
  margin-bottom: 20px;
`

const MobileImageWrap = styled.div`
  align-items: center;
  display: flex;
  justify-content: center;
`

const MobileImage = styled.img`
  margin-bottom: 20px;
  width: 50%;
`

const Home = () => {
  return (
    <Query query={HomePageQuery}>
      {({ loading, data: { pageStart } }: Props) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={pageStart.headerImage}
              messageBgColor={pageStart.headerTextBgColor}
              messageOne={pageStart.headerText1}
              messageTwo={pageStart.headerText2}
            />

            <HomeGridColumn>
              <Content>
                <StylizedIteam src={iteamStylized} />
                <Texts>
                  <TextBlock>
                    <MediaQuery maxDeviceWidth={480}>
                      <MobileImageWrap>
                        <MobileImage src={pageStart.codeMobileImage} />
                      </MobileImageWrap>
                    </MediaQuery>
                    <H1>{pageStart.codeTitle}</H1>
                    <Paragraph>{pageStart.codeText}</Paragraph>
                    <CompanyLogo src={pageStart.codeLogo} />
                    <Link to={`/case/${pageStart.codeSlug}`}>
                      {pageStart.codeLinkText}
                    </Link>
                  </TextBlock>

                  <TextBlock>
                    <MediaQuery maxDeviceWidth={480}>
                      <MobileImageWrap>
                        <MobileImage src={pageStart.strategyMobileImage} />
                      </MobileImageWrap>
                    </MediaQuery>
                    <H1>{pageStart.strategyTitle}</H1>
                    <Paragraph>{pageStart.strategyText}</Paragraph>{' '}
                    <CompanyLogo src={pageStart.strategyLogo} />
                    <Link to={`/case/${pageStart.strategySlug}`}>
                      {pageStart.strategyLinkText}
                    </Link>
                  </TextBlock>

                  <TextBlock>
                    <MediaQuery maxDeviceWidth={480}>
                      <MobileImageWrap>
                        <MobileImage src={pageStart.cultureMobileImage} />
                      </MobileImageWrap>
                    </MediaQuery>
                    <H1>{pageStart.cultureTitle}</H1>
                    <Paragraph>{pageStart.cultureText}</Paragraph>{' '}
                    <CompanyLogo src={pageStart.cultureLogo} />
                    <Link to={`/case/${pageStart.cultureSlug}`}>
                      {pageStart.cultureLinkText}
                    </Link>
                  </TextBlock>
                </Texts>
              </Content>
            </HomeGridColumn>

            <OperationsBanner />
            <Team shortName={['met', 'rln', 'msr', 'jmn']} />
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default Home
