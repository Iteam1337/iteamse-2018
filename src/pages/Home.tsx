import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import Helmet from 'react-helmet-async'
import { StartQuery } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import ImageBlock from '../components/Blocks/ImageBlock'
import GridColumn from '../components/Grid/GridColumn'
import Team from '../components/Team/Team'
import howWeWorkImage from './img/illustrations-group-3.svg'

export const HOME_PAGE_QUERY = gql`
  query Start {
    pageStart2019 {
      team
      weAreTitle
      weAreText
      weOfferTitle
      weOfferText
      ourMethodTitle
      ourMethodText
    }
  }
`

class HomeQuery extends Query<StartQuery> {}

const Home = () => {
  return (
    <>
      <Helmet>
        <title>Iteam - There's a better way</title>
        <meta property="og:title" content="Iteam - There's a better way" />
        <meta property="twitter:title" content="Iteam - There's a better way" />
      </Helmet>

      <HomeQuery query={HOME_PAGE_QUERY}>
        {({ loading, data }) => {
          if (loading || !data) {
            return null
          }

          const { pageStart2019 } = data

          return (
            <>
              <GridColumn>
                <Block title={pageStart2019.weAreTitle}>
                  {pageStart2019.weAreText}
                </Block>
                <Block title={pageStart2019.weOfferTitle}>
                  {pageStart2019.weOfferText}
                </Block>
                <ImageBlock image={howWeWorkImage} />
                <Block title={pageStart2019.ourMethodTitle}>
                  {pageStart2019.ourMethodText}
                </Block>
              </GridColumn>
              <Team shortName={pageStart2019.team} />
            </>
          )
        }}
      </HomeQuery>
    </>
  )
}

export default Home
