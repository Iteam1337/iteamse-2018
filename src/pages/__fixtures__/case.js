export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
