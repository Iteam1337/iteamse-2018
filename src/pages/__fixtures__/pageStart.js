export const pageStart = {
  __typename: 'PageStart',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/6WZT7iYMZGkuEUIc4S0KgK/499955f736b51dfde591a5195803a445/start_optimized.jpg',
  headerText1: 'Vi digitaliserar företag och organisationer',
  headerText2: 'genom strategi, kod och kultur',
  headerTextBgColor: '#FF3B5C',
  codeMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2mxfqVCqiw4CQYguMSYwYU/b9494a7ffcc913a275984ba7db02a1bf/dator.png',
  codeText:
    'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med.',
  codeTitle: 'Kod',
  codeLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/3qnZSfySW4cEGoUo6KCAae/8a656eba15e15b61dacc9fbea38aa4bd/logo_af.png',
  codeSlug: 'arbetsformedlingen',
  codeLinkText: 'Läs om hur vi hjälper Arbetsförmedlingen med kod',
  cultureMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2y9WBlATsssIcSQmuQ6Ky8/32e27eb88cb9d3063be20803442e3a02/chat.png',
  cultureText:
    'Ibland ser vi oss som kulturarbetare. Det räcker inte att bygga digitala tjänster åt sina användare. Det är först när hela din organisation tänker, andas och lever digitalt som ni digitaliseras. på riktigt. Vi rustar dig för framtiden helt enkelt.',
  cultureTitle: 'Kultur',
  cultureLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/4D844WR1v20ICIOG4SK0y8/15a008277f5565d9f52fbd5312f6e121/logo_seb.png',
  cultureSlug: 'seb',
  cultureLinkText: 'Läs om hur vi hjälper SEB med kultur',
  strategyMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/1oqMfeZBnuQAAGweIGCKEW/d34548ee301d4d3f315e2ee313089093/Iteam-03-e.png',
  strategyText:
    'Vi digitaliserar Sverige. Pompigt! Men det betyder att vi hjälper dig att leda digitaliseringen istället för att vara den som blir lämnad på efterkälken. Både på den marknad där du verkar, och de marknader du inte ens visste att du borde verka på.',
  strategyTitle: 'Strategi',
  strategyLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/1mv5fxBGLmeyWom4OI2WsI/a4d79cae2076b43aea926db2f1244b52/logo_tsab.png',
  strategySlug: 'tsab',
  strategyLinkText: 'Läs om hur vi hjälper Taxi Stockholm med strategi',
}
