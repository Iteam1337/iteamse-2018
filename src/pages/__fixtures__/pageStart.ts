export const pageStart = {
  __typename: 'PageStart',
  codeLinkText: 'Läs om hur vi hjälper Arbetsförmedlingen med kod',
  codeLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/3qnZSfySW4cEGoUo6KCAae/8a656eba15e15b61dacc9fbea38aa4bd/logo_af.png',
  codeMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2mxfqVCqiw4CQYguMSYwYU/b9494a7ffcc913a275984ba7db02a1bf/dator.png',
  codeSlug: 'arbetsformedlingen',
  codeText:
    'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med.',
  codeTitle: 'Kod',
  contactTitle: 'Hör av dig till oss!',
  cultureLinkText: 'Läs om hur vi hjälper SEB med kultur',
  cultureLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/4D844WR1v20ICIOG4SK0y8/15a008277f5565d9f52fbd5312f6e121/logo_seb.png',
  cultureMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2y9WBlATsssIcSQmuQ6Ky8/32e27eb88cb9d3063be20803442e3a02/chat.png',
  cultureSlug: 'seb',
  cultureText:
    'Ibland ser vi oss som kulturarbetare. Det räcker inte att bygga digitala tjänster åt sina användare. Det är först när hela din organisation tänker, andas och lever digitalt som ni digitaliseras. på riktigt. Vi rustar dig för framtiden helt enkelt.',
  cultureTitle: 'Kultur',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/6WZT7iYMZGkuEUIc4S0KgK/499955f736b51dfde591a5195803a445/start_optimized.jpg',
  headerText1: 'Vi digitaliserar företag och organisationer',
  headerText2: 'genom strategi, kod och kultur',
  headerTextBgColor: '#FF3B5C',
  strategyLinkText: 'Läs om hur vi hjälper Taxi Stockholm med strategi',
  strategyLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/1mv5fxBGLmeyWom4OI2WsI/a4d79cae2076b43aea926db2f1244b52/logo_tsab.png',
  strategyMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/1oqMfeZBnuQAAGweIGCKEW/d34548ee301d4d3f315e2ee313089093/Iteam-03-e.png',
  strategySlug: 'tsab',
  strategyText:
    'Vi digitaliserar Sverige. Pompigt! Men det betyder att vi hjälper dig att leda digitaliseringen istället för att vara den som blir lämnad på efterkälken. Både på den marknad där du verkar, och de marknader du inte ens visste att du borde verka på.',
  strategyTitle: 'Strategi',
  team: ['rln', 'msr'],
}
