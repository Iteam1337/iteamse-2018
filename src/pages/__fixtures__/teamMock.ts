import { TEAM_QUERY } from '../../components/Team/Team'
import { teamMembers } from './teamMembers'

export const TeamQueryMock = {
  request: {
    query: TEAM_QUERY,
  },
  result: {
    data: {
      teamMembers,
    },
  },
}
