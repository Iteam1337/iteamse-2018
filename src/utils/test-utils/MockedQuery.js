import * as React from 'react'
import { ApolloProvider } from 'react-apollo'
import ApolloClient from 'apollo-client'
import { createHttpLink } from 'apollo-link-http'
import createMockedNetworkFetch from 'apollo-mocknetworkinterface'
import { InMemoryCache } from 'apollo-cache-inmemory'
import { BrowserRouter } from 'react-router-dom'
import { ThemeProvider } from 'styled-components'
import { theme } from '../../theme'

type P = {
  children: React.Node,
  loading?: boolean,
  response: Object,
}

const MockedQuery = ({ children, loading, response = {} }: P) => {
  const createResponse = () => {
    return {
      loading: !!loading,
      data: {
        ...response,
      },
    }
  }

  const mockedNetworkFetch = createMockedNetworkFetch(createResponse, {
    latency: 0,
  })

  const client = new ApolloClient({
    link: createHttpLink({
      uri: 'http://localhost:3000',
      fetch: mockedNetworkFetch,
    }),
    cache: new InMemoryCache(),
  })

  return (
    <ApolloProvider client={client}>
      <ThemeProvider theme={theme}>
        <BrowserRouter>{children}</BrowserRouter>
      </ThemeProvider>
    </ApolloProvider>
  )
}

export default MockedQuery
