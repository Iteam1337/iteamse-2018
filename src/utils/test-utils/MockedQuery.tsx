import { InMemoryCache } from 'apollo-cache-inmemory'
import ApolloClient from 'apollo-client'
import * as React from 'react'
import { ApolloProvider } from 'react-apollo'
import { mockSingleLink } from 'react-apollo/test-utils'
import MockRouter from 'react-mock-router'
import { ThemeProvider } from 'styled-components'
import { theme } from '../../theme'

interface MockedQueryProps {
  location?: object
  mocks: any[]
  params?: object
}

const MockedQuery: React.SFC<MockedQueryProps> = ({
  children,
  location,
  mocks,
  params,
}) => {
  const link = mockSingleLink.apply(null, mocks)

  const client = new ApolloClient({
    cache: new InMemoryCache(),
    link,
  })

  return (
    <ApolloProvider client={client}>
      <ThemeProvider theme={theme}>
        <MockRouter location={location} params={params}>
          {children}
        </MockRouter>
      </ThemeProvider>
    </ApolloProvider>
  )
}

export default MockedQuery
