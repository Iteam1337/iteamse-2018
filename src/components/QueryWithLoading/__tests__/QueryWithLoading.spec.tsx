import React from 'react'
import { render, wait } from 'react-testing-library'

import { HomePageQuery } from '../../../../typings/iteamse'
import { pageStart } from '../../../pages/__fixtures__/pageStart'
import { TeamQueryMock } from '../../../pages/__fixtures__/teamMock'
import { HOME_PAGE_QUERY } from '../../../pages/Home'
import MockedQuery from '../../../utils/test-utils/MockedQuery'

import { Query } from 'react-apollo'
import queryWithLoading from '../QueryWithLoading'

const mocks = [
  {
    request: {
      query: HOME_PAGE_QUERY,
    },
    result: {
      data: {
        pageStart,
      },
    },
  },
  TeamQueryMock,
]

describe('components/QueryWithLoading', () => {
  it('renders', async () => {
    class HomeQuery extends Query<HomePageQuery> {}
    const HomeQueryWithLoading = queryWithLoading(HomeQuery)
    const { container } = render(
      <MockedQuery mocks={mocks}>
        <HomeQueryWithLoading query={HOME_PAGE_QUERY}>
          {data => <pre>{JSON.stringify(data, null, 2)}</pre>}
        </HomeQueryWithLoading>
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })

  it('renders loading indicator when passed no data', async () => {
    class HomeQuery extends Query<HomePageQuery> {}
    const HomeQueryWithLoading = queryWithLoading(HomeQuery)
    const { container } = render(
      <MockedQuery mocks={[]}>
        <HomeQueryWithLoading query={HOME_PAGE_QUERY}>
          {data => <pre>{JSON.stringify(data, null, 2)}</pre>}
        </HomeQueryWithLoading>
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
