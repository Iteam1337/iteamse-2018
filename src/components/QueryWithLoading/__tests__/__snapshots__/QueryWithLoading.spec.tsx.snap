// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/QueryWithLoading renders 1`] = `
<div>
  <pre>
    { "pageStart": { "headerImage": "//images.ctfassets.net/rj4r6yfcesw5/6WZT7iYMZGkuEUIc4S0KgK/499955f736b51dfde591a5195803a445/start_optimized.jpg", "headerText1": "Vi digitaliserar företag och organisationer", "headerText2": "genom strategi, kod och kultur", "headerTextBgColor": "#FF3B5C", "aiTitle": "Utforska möjligheterna med AI", "aiText": "We are offering an Artificial Intelligence package to help you build AI products and services. We have a very competent Data Science team that will help you grow in the growing market of Artificial Intelligence.", "aiSlug": "erbjudanden/ai", "aiLinkText": "Läs mer om våra erbjudanden inom AI", "codeMobileImage": "//images.ctfassets.net/rj4r6yfcesw5/2mxfqVCqiw4CQYguMSYwYU/b9494a7ffcc913a275984ba7db02a1bf/dator.png", "codeText": "Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med.", "codeTitle": "Kod", "codeLogo": "//images.ctfassets.net/rj4r6yfcesw5/3qnZSfySW4cEGoUo6KCAae/8a656eba15e15b61dacc9fbea38aa4bd/logo_af.png", "codeSlug": "arbetsformedlingen", "codeLinkText": "Läs om hur vi hjälper Arbetsförmedlingen med kod", "contactTitle": "Hör av dig till oss!", "cultureMobileImage": "//images.ctfassets.net/rj4r6yfcesw5/2y9WBlATsssIcSQmuQ6Ky8/32e27eb88cb9d3063be20803442e3a02/chat.png", "cultureText": "Ibland ser vi oss som kulturarbetare. Det räcker inte att bygga digitala tjänster åt sina användare. Det är först när hela din organisation tänker, andas och lever digitalt som ni digitaliseras. på riktigt. Vi rustar dig för framtiden helt enkelt.", "cultureTitle": "Kultur", "cultureLogo": "//images.ctfassets.net/rj4r6yfcesw5/4D844WR1v20ICIOG4SK0y8/15a008277f5565d9f52fbd5312f6e121/logo_seb.png", "cultureSlug": "seb", "cultureLinkText": "Läs om hur vi hjälper SEB med kultur", "strategyMobileImage": "//images.ctfassets.net/rj4r6yfcesw5/1oqMfeZBnuQAAGweIGCKEW/d34548ee301d4d3f315e2ee313089093/Iteam-03-e.png", "strategyText": "Vi digitaliserar Sverige. Pompigt! Men det betyder att vi hjälper dig att leda digitaliseringen istället för att vara den som blir lämnad på efterkälken. Både på den marknad där du verkar, och de marknader du inte ens visste att du borde verka på.", "strategyTitle": "Strategi", "strategyLogo": "//images.ctfassets.net/rj4r6yfcesw5/1mv5fxBGLmeyWom4OI2WsI/a4d79cae2076b43aea926db2f1244b52/logo_tsab.png", "strategySlug": "tsab", "strategyLinkText": "Läs om hur vi hjälper Taxi Stockholm med strategi", "team": [ "rln", "msr" ], "__typename": "PageStart" } }
  </pre>
</div>
`;

exports[`components/QueryWithLoading renders loading indicator when passed no data 1`] = `
<div>
  <div
    class="sc-VigVT xMFhY sc-gzVnrw bIbQZT"
  >
    <div
      class="sc-jTzLTM hsHbNM"
    />
  </div>
  <div
    class="sc-fjdhpX dSePDG sc-htoDjs bjndSh"
  >
    <div
      class="sc-jzJRlG eDqLMO"
      width="100"
    />
    <div
      class="sc-jzJRlG eDqLMO"
      width="100"
    />
    <div
      class="sc-jzJRlG fBYRlq"
      width="75"
    />
  </div>
</div>
`;
